applications/polyp_detection/README.md

```diff
@@ -0,0 +1,53 @@
+# Polyp Detection
+
+This application demonstrates how to run polyp detection models on live video in real-time.
+
+The model: [RT-DETR](https://github.com/lyuwenyu/RT-DETR) is trained on the [REAL-Colon](https://www.nature.com/articles/s41597-024-03359-0) dataset.
+
+## Setup Instructions
+
+### Build Docker and Run
+
+```Bash
+docker build -t holoscan:polyp-det -f Dockerfile .
+```
+
+Please mount the directory that contains `holohub/` into `/colon_workspace`. For example, my folder is under `/raid/colon_workspace`:
+
+```Bash
+docker run --rm -it --gpus=all --ipc=host -v /raid/colon_workspace:/colon_workspace holoscan:polyp-det
+```
+
+### Get ONNX model
+
+(TODO) need to upload after license permission
```

Comment:
Hi @jjomier @MostafaToloui , for the model weights, may I know if there are any requirements on where to upload it? Like on NGC or somewhere else? Thanks!